packages/cli/test/functional/test_site/expected/sub_site/nested_sub_site/testNunjucksPathResolving.html

```diff
@@ -213,7 +213,7 @@ <h3 id="testing-site-nav"><span id="testing-site-nav" class="anchor"></span>Test
           The root directory in this case is the respective root directory of the root site or sub sites.</p>
         <hr>
         <p><strong>Test {% include %}</strong></p>
-        <p>This is content from testNunjucksPathResolvingInclude.mbdf located at <code class="line-numbers hljs inline no-lang" v-pre>sub_site/nested_sub_site</code></p>
+        <p>This is content from <a href="http://testNunjucksPathResolvingInclude.md">testNunjucksPathResolvingInclude.md</a> located at <code class="line-numbers hljs inline no-lang" v-pre>sub_site/nested_sub_site</code></p>
```

Comment:
Looks like a bug here? This shouldn't be wrapped in `<a>` and the `href` attribute does not make sense.